src/engraving/playback/playbackcontext.cpp

```diff
@@ -93,18 +93,18 @@ DynamicLevelMap PlaybackContext::dynamicLevelMap(const Score* score) const
     }
 
     if (result.empty()) {
-        result.emplace(0, mpe::dynamicLevelFromType(mpe::DynamicType::Natural));
+        result.emplace(0, mpe::dynamicLevelFromType(mpe::DynamicType::mp) * 0.5 + mpe::dynamicLevelFromType(mpe::DynamicType::mf) * 0.5);
```

Comment:
If no dynamics are in the score, we go ahead and play back at halfway between "mp" and "mf".  May want to just set this to "mf", though.